tests/unit/test_client.py

```diff
@@ -19,3 +19,13 @@
 def test_parse_project_and_dataset(project, dataset, expected):
     got = ibis_bigquery.client.parse_project_and_dataset(project, dataset)
     assert got == expected
+
+
+def test_parse_project_and_dataset_raises_error():
+    with pytest.raises(ValueError) as exception:
+        ibis_bigquery.client.parse_project_and_dataset(
+            "my-project", "data-project.my_dataset.table"
+        )
+    assert "data-project.my_dataset.table is not a BigQuery dataset" in str(
```

Comment:
Typically this is done with `pytest.raises(..., match="some string")`. Any reason not to do that here?

Reply:
Fixed. I didn't know about the `match` argument.